### PR TITLE
Fix dashboard profile validation

### PR DESCRIPTION
**Pull request by zaelgohary**
### Description

- Remove auto focus to disable validation unless inputs are entered
- Close profile modal once connected/logged in

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/860
- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/856

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Comment by AhmedHanafy725** [comment]
the QR code should be shown after logging in like the playground

**Comment by zaelgohary** [comment]
> the QR code should be shown after logging in like the playground

Added.

File: packages/dashboard/src/components/TfChainConnector.vue [hunk 1]
```diff
@@ -224,8 +246,7 @@ enum SSHState {
   },
 })
 export default class TfChainConnector extends Vue {
-  /* devnet bridge for now */
-  public bridge = "GDHJP6TF3UXYXTNEZ2P36J5FH7W4BJJQ4AYYAXC66I2Q2AH5B6O6BCFG";
+  public bridge = (window as any).configs.APP_BRIDGE_TFT_ADDRESS;
```
Comment (AhmedHanafy725) [inline packages/dashboard/src/components/TfChainConnector.vue:249]: can we use the value [here](https://github.com/threefoldtech/tfgrid-sdk-ts/blob/development/packages/dashboard/src/portal/config.ts#L5) in the config file?

Comment (zaelgohary) [reply packages/dashboard/src/components/TfChainConnector.vue]: done.

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**